submissions/ksalpern/Hooli-style-Popup/scss/style.scss

```diff
@@ -0,0 +1,280 @@
+html {
+  box-sizing: border-box;
+  height: 100%;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+  padding: 0;
+  margin: 0;
+}
+body {
+  min-width: 320px;
+  overflow-x: hidden;
+  height: 100%;
+}
+ol,
+ul {
+  list-style: none;
+}
+a,
+a:link,
+a:visited,
+a:hover {
+  text-decoration: none;
+}
+img {
+  max-width: 40px;
+  height: auto;
+  vertical-align: top;
+}
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(212, 240, 231);
+  padding: 24px 0px;
+}
+
+.header {
+  .header__wrapper {
```

Comment:
Avoid unnecessary nesting in CSS. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) about it